statics/index.html

```diff
@@ -146,6 +149,9 @@
             <router-link v-if="service == 'Analyzer'" tag="li" to="/discovery">
               <a>Discovery</a>
             </router-link>
+            <router-link tag="li" to="/status">
```

Comment:
I would prefer to have the tab at the top right (at the left of "Logout" for instance)

Reply:
done